app/src/main/java/com/xmartlabs/template/service/adapter/EpochSecondsLocalDateAdapter.java

```diff
@@ -0,0 +1,45 @@
+package com.xmartlabs.template.service.adapter;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import com.xmartlabs.template.helper.StringUtils;
+
+import org.threeten.bp.LocalDate;
+import org.threeten.bp.LocalDateTime;
+import org.threeten.bp.ZoneOffset;
+
+import java.lang.reflect.Type;
+
+import timber.log.Timber;
+
+/**
+ * Created by mirland on 24/08/16.
+ */
+public class EpochSecondsLocalDateAdapter implements JsonSerializer<LocalDate>, JsonDeserializer<LocalDate> {
```

Comment:
Yeah but seconds is never used. I think we should just leave a milliseconds one.